front/src/common/api/generatedEditoastApi.ts

```diff
@@ -3310,7 +3311,97 @@ export type TrainSchedule = {
   start_time: string;
   train_name: string;
 };
+export type CategoryChangeGroup = {
+  value: string;
+};
+export type ConstraintDistributionChangeGroup = {
+  value: Distribution;
+};
+export type InitialSpeedChangeGroup = {
+  value: number;
+};
+export type LabelsChangeGroup = {
+  value: string[];
+};
+export type TrainScheduleOptions = {
+  use_electrical_profiles?: boolean;
+  use_speed_limits_for_simulation?: boolean;
+};
+export type OptionsChangeGroup = {
+  value: TrainScheduleOptions;
+};
+export type Margins = {
+  boundaries: string[];
+  /** The values of the margins. Must contains one more element than the boundaries
+    Can be a percentage `X%` or a time in minutes per 100 kilometer `Xmin/100km` */
+  values: string[];
+};
+export type PathItem = PathItemLocation & {
+  /** Metadata given to mark a point as wishing to be deleted by the user.
+    It's useful for soft deleting the point (waiting to fix / remove all references)
+    If true, the train schedule is consider as invalid and must be edited */
+  deleted?: boolean;
+  id: string;
+};
+export type PowerRestrictionItem = {
+  from: string;
+  to: string;
+  value: string;
+};
+export type ScheduleItem = {
+  /** The expected arrival time at the stop.
+    This will be used to compute the final simulation time. */
+  arrival?: string | null;
+  at: string;
+  /** Whether the schedule item is locked (only for display purposes) */
+  locked?: boolean;
+  reception_signal?: ReceptionSignal;
+  /** Duration of the stop.
+    Can be `None` if the train does not stop.
+    If `None`, `reception_signal` must be `Open`.
+    `Some("PT0S")` means the train stops for 0 seconds. */
+  stop_for?: string | null;
+};
+export type PathAndScheduleChangeGroup = {
+  margins: Margins;
+  path: PathItem[];
+  power_restrictions: PowerRestrictionItem[];
+  schedule: ScheduleItem[];
+};
+export type RollingStockChangeGroup = {
+  category?: TrainCategory | null;
+  comfort: Comfort;
+  rolling_stock_name: string;
+};
+export type StartTimeChangeGroup = {
+  value: string;
+};
+export type TrainNameChangeGroup = {
+  value: string;
+};
+export type PacedTrainException = (
+  | (object | null)
+  | {
+      disabled: boolean;
+      occurence_index: number;
+    }
+) & {
+  category?: CategoryChangeGroup | null;
+  constraint_distribution?: ConstraintDistributionChangeGroup | null;
+  initial_speed?: InitialSpeedChangeGroup | null;
+  labels?: LabelsChangeGroup | null;
+  options?: OptionsChangeGroup | null;
+  path_and_schedule?: PathAndScheduleChangeGroup | null;
+  rolling_stock?: RollingStockChangeGroup | null;
+  speed_limit_tag?: {
+    value?: string | null;
+  } | null;
```

Comment:
Any reason why we don't have a `ChangeGroup` type for this one, given we have one for each other field?